src/Drinks.js

```diff
@@ -0,0 +1,47 @@
+// Drinks Component
+
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+import AllIngredients from "./AllIngredients.js";
+
+function Drinks({userInput}) {
+
+    const [drinkResults, setDrinkResults] = useState([]);
+
+    useEffect(() => {
+        axios({
+            url: "https://www.thecocktaildb.com/api/json/v1/1/search.php",
+            method: "GET",
+            dataResponse: "json",
+            params: {
+                s: userInput,
+            }
+        }).then((response) => {
+            setDrinkResults(response.data.drinks);            
+        }).catch(err => {
+            console.log(err);
+        })
+    }, [userInput] )
+
+    return(
```

Comment:
if there is an error, for example, when writing, it shows a white screen, I would put a message, for the user to write again
I have to refresh the page